core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -29,18 +28,12 @@
 
     <div class="conversation-skin-cards-container conversation-skin-animate-cards"
          ng-class="{'animate-to-two-cards': isAnimatingToTwoCards, 'animate-to-one-card': isAnimatingToOneCard}">
-
-      <!--
-        If the interaction is not inline, two cases arise: (a) the screen is
-        narrow, in which case only one card shows on the screen at a time, and
-        a selection bar appears at the bottom of the screen to switch between
-        cards, (b) the screen is wide, in which case both cards appear
-        side-by-side.
-      -->
-      <div class="conversation-skin-main-tutor-card conversation-skin-tutor-card-active"
-          ng-class="{'conversation-skin-left-card': isCurrentSupplementalCardNonempty() && canWindowFitTwoCards(), 'conversation-skin-has-subsidiary-cards': isCurrentSupplementalCardNonempty() && !canWindowFitTwoCards(),
-            'animate-card-width': startCardChangeAnimation}"
-          ng-show="isPanelVisible(PANEL_TUTOR)">
+      <div ng-show='!isViewportNarrow() || isScreenNarrowAndShowingTutorCard()'
+           class="conversation-skin-main-tutor-card"
+           ng-class="{'animate-card-width': startCardChangeAnimation,
+                      'tutor-card-shadow': isViewportNarrow() && isCurrentSupplementalCardNonempty(),
```

Comment:
I think, let's call this class 'has-shadow', since the CSS definition doesn't seem specific to the tutor card.